dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/categories/environmental/subcategories/SfdrEnvironmentalGreenhouseGasEmissions.kt

```diff
@@ -15,11 +15,27 @@ data class SfdrEnvironmentalGreenhouseGasEmissions(
 
     val scope2GhgEmissionsInTonnes: ExtendedDataPoint<BigDecimal>? = null,
 
+    val scope2GhgEmissionsLocationBasedInTonnes: ExtendedDataPoint<BigDecimal>? = null,
+
+    val scope2GhgEmissionsMarketBasedInTonnes: ExtendedDataPoint<BigDecimal>? = null,
+
+    val scope1And2GhgEmissionsInTonnes: ExtendedDataPoint<BigDecimal>? = null,
+
+    val scope1And2GhgEmissionsLocationBasedInTonnes: ExtendedDataPoint<BigDecimal>? = null,
+
+    val scope1And2GhgEmissionsMarketBasedInTonnes: ExtendedDataPoint<BigDecimal>? = null,
```

Comment:
This is because we basse on the "additional fields for sfdr" branch to also include them in ours. Now the "additional fields" branch is already merged to main